File: app/src/main/java/studio/mandysa/music/logic/Model/PlaylistInfoModel.java
package studio.mandysa.music.logic.Model;

import java.util.List;

import mandysax.anna2.annotation.Value;

public class PlaylistInfoModel {
    @Value("name")
    public String name;

    @Value("description")
    public String description;

    @Value("coverImgUrl")
    public String coverImgUrl;

    @Value("trackIds")
    public List<SongList> songList;

    public static class SongList {

        @Value("id")
        public String id;

    }
}

File: app/src/main/java/studio/mandysa/music/logic/model/PlaylistInfoModel.kt
package studio.mandysa.music.logic.model

import mandysax.anna2.annotation.Value

/**
 * @author liuxiaoliu66
 */
class PlaylistInfoModel {
    @Value("name")
    var name: String? = null

    @Value("description")
    var description: String? = null

    @Value("coverImgUrl")
    var coverImgUrl: String? = null

    @Value("trackIds")
    var songList: List<SongList>? = null

    class SongList {
        @Value("id")
        var id: String? = null

        override fun toString(): String {
            return id!!
        }
    }
}